### PR TITLE
utils_windows: Add file existing check function for windows

### PR DESCRIPTION
**Pull request by lijinlijin**
Checking if a file is existing in windows guest, reture true if yes.

ID: 1455812

Signed-off-by: Li Jin <lijin@redhat.com>

### CODE REVIEW

**Comment by lijinlijin** [comment]
@peixiu @xiagao 
Could you help to review this PR?

Thanks

File: virttest/utils_windows/system.py [hunk 1]
```diff
@@ -44,3 +44,19 @@ def os_arch(session):
...
+def file_existing(session, filename):
+    """
+    Check if a file is existing.
+
+    :param session: Session object.
+    :param filename: File name with full path.
+
+    :return: bool value: True if file exists.
+    """
+    check_cmd = "type %s" % filename
```
Comment (luckyh) [inline virttest/utils_windows/system.py]: Can we replace it with `dir`?

Comment (lijinlijin) [reply virttest/utils_windows/system.py]: Sure, updated.

Thanks

File: virttest/utils_windows/system.py [hunk 2]
```diff
@@ -44,3 +44,19 @@ def os_arch(session):
...
     return _osinfo(session, props=["OSArchitecture"])
+
+
+def file_existing(session, filename):
+    """
+    Check if a file is existing.
+
+    :param session: Session object.
+    :param filename: File name with full path.
+
```
Comment (xiagao) [inline virttest/utils_windows/system.py]: Seems there is no empty line for this docstring type.
Others is ok for me.

**Comment by peixiu** [comment]
LGTM~
Acked-by: Peixiu Hou phou@redhat.com

File: virttest/utils_windows/system.py [hunk 3]
```diff
@@ -44,3 +44,18 @@ def os_arch(session):
...
+def file_existing(session, filename):
+    """
+    Check if a file is existing.
+
+    :param session: Session object.
+    :param filename: File name with full path.
+    :return: bool value: True if file exists.
+    """
+    check_cmd = "dir %s" % filename
```
Comment (luckyh) [inline virttest/utils_windows/system.py]: To handle hidden files and systems files the command should be appended with option `/a`
```
C:\>dir c:\Users\Administrator\Desktop\desktop.ini
 Volume in drive C is Windows
 Volume Serial Number is 8C27-4256

 Directory of c:\Users\Administrator\Desktop

File Not Found

```
```
C:\>dir /a c:\Users\Administrator\Desktop\desktop.ini
 Volume in drive C is Windows
 Volume Serial Number is 8C27-4256

 Directory of c:\Users\Administrator\Desktop

07/27/2018  03:42 AM               282 desktop.ini
               1 File(s)            282 bytes
               0 Dir(s)  16,444,526,592 bytes free

```
Also recommend adding option `/b` to reduce log size.

File: virttest/utils_windows/system.py [hunk 4]
```diff
@@ -44,3 +44,18 @@ def os_arch(session):
     :return: Windows OS architecture.
     """
     return _osinfo(session, props=["OSArchitecture"])
+
+
+def file_existing(session, filename):
```
Comment (hereischen) [inline virttest/utils_windows/system.py]: `file_exists(session, filename)`
Please note, all my comments are suggestions.

File: virttest/utils_windows/system.py [hunk 5]
```diff
@@ -44,3 +44,18 @@ def os_arch(session):
...
+    """
+    Check if a file is existing.
```
Comment (hereischen) [inline virttest/utils_windows/system.py]: Check if a file exists.

File: virttest/utils_windows/system.py [hunk 6]
```diff
@@ -44,3 +44,18 @@ def os_arch(session):
...
+
+    :param session: Session object.
+    :param filename: File name with full path.
+    :return: bool value: True if file exists.
+    """
+    check_cmd = "dir /a /b %s" % filename
+    if not session.cmd_status(check_cmd):
+        return True
+    else:
```
Comment (hereischen) [inline virttest/utils_windows/system.py]:
```python 
 if not session.cmd_status(check_cmd):
      return True
  return False
```

**Pull request merged**